.travis.yml

```diff
@@ -11,7 +11,7 @@ android:
     - android-30
 
 script:
-  - ./gradlew clean build
+  - ./gradlew clean ktlintCheck build
```

Comment:
ktlint should be placed in separate "job". 
As a first improvement place it in a separate "stage" which will be executed before rest of the stages:
Configuration to be created:
- first stage - should run "ktlint",
- second stage - should run "build".

Of course name stages accordingly :)